install/kubernetes/versions.json

```diff
@@ -0,0 +1,8 @@
+{
+  "version": "1.8.90",
```

Comment:
Ideally this specific version should be specified by the contents of the `VERSION` file at the top of the repo, not duplicated here (otherwise we still need to edit this file or generate it from the source of truth, the `VERSION` file).

The only reason for the definition in the Makefile recently was to work around some issues before we branched for v1.9, so the latest version should remove that definition there and revert back to using the `VERSION` file.